tests/Standalone/ClassWithDifferentCodeStyleTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\Tests\Standalone;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Core\Standalone\RectorStandaloneRunnerStaticFactory;
+
+/**
+ * Class ClassWithDifferentCodeStyleTest
+ *
+ * We expect that Rector will not make changes to the code style for things it doesn't touch.
+ * In this example, the `{` should not be moved.
+ */
+final class ClassWithDifferentCodeStyleTest extends TestCase
+{
+    public function test(): void
+    {
+        $rectorStandaloneRunner = RectorStandaloneRunnerStaticFactory::create();
```

Comment:
This static factory is designed for extreme/legacy/weird cases where rector cannot be run via command line.

Better use this approach:
https://github.com/rectorphp/rector/tree/master/tests/Issues